datalad/downloaders/base.py

```diff
@@ -176,6 +176,12 @@ def access(self, method, url, allow_old_session=True, **kwargs):
                 break
             except AccessDeniedError as e:
                 ce = CapturedException(e)
+                if hasattr(e, 'status') and e.status == 429:
+                    # Too many requests.
+                    # We can retry by continuing the loop.
+                    time.sleep(0.5)
```

Comment:
```suggestion
                    time.sleep(0.5*(attempt**1.2))
```
some exponential growth is good here ;)